integration/Jenkinsfile

```diff
@@ -111,25 +106,28 @@ pipeline {
                                 '''
                             }
                         }
-
                         stage('Install RPM pkg') {
                             steps {
                                 sh '''
                                 echo "Installing"
-                                export PATH=$PATH:/usr/local/go/bin
+                                export BUILD_PATH=https://github.com/NetApp/harvest/releases/download/v21.08.0/harvest-21.08.0-6.x86_64.rpm
```

Comment:
downloading from github will cause our metrics to skewed @ https://hanadigital.github.io/grev/?user=NetApp&repo=harvest

May be copy it locally?

Reply:
good catch. fixed.